package/MDAnalysis/coordinates/PQR.py

```diff
@@ -128,7 +128,7 @@ def _read_first_frame(self):
             for line in pqrfile:
                 if line.startswith(('ATOM', 'HETATM')):
                     fields = line.split()
-                    coords.append(map(float, fields[-5:-2]))
+                    coords.append(np.float32(fields[-5:-2]))
```

Comment:
This depends on the exact workload. But yeah the list comprehensions seems to be marginally faster